backend/modules/query_controllers/multimodal/types.py

```diff
@@ -155,3 +156,13 @@ def validate_retriever_type(cls, values: Dict) -> Dict:
             )
 
         return values
+
+
+class Answer(BaseModel):
+    type: str = "answer"
+    content: str
+
+
+class Docs(BaseModel):
+    type: str = "docs"
+    content: Optional[List[Document]] = None
```

Comment:
Is nullability important?

```suggestion
    content: List[Document] = Field(default_factory=list)
```